Formula/nerd-fonts-patcher.rb

```diff
@@ -0,0 +1,28 @@
+class NerdFontsPatcher < Formula
+  desc "Patch your own font"
+  homepage "https://github.com/ryanoasis/nerd-fonts#font-patcher"
+  url "https://github.com/ryanoasis/nerd-fonts/trunk/src", using: :svn
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#we-dont-like-install-scripts-that-download-unversioned-things